### PR TITLE
Add integration tests -- RocketGate

### PR DESCRIPTION
**Pull request by rlcooper46**
Adds integration tests for rocketgate

### CODE REVIEW

File: gateways/rocketgate/request_builders.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func buildAuthRequest(merchantID string, merchantPassword string, merchantAccoun
 	}
 
 	// Ignore CVV and AVS check
-	gatewayRequest.Set(request.CVV2_CHECK, "IGNORE")
+	gatewayRequest.Set(request.CVV2_CHECK, "NO")
 	gatewayRequest.Set(request.AVS_CHECK, "IGNORE")
 
 	return gatewayRequest
```
Comment (yuweihsu01) [inline gateways/rocketgate/request_builders.go:53]: I cannot comment on those lines, but for `buildCaptureRequest` and `buildRefundRequest`, you also need to change to 
`gatewayRequest.Set(request.AMOUNT, sleet.AmountToDecimalString(captureRequest.Amount))`
`gatewayRequest.Set(request.AMOUNT, sleet.AmountToDecimalString(refundRequest.Amount))`

File: integration-tests/rocketgate_test.go [hunk 2]
```diff
@@ -1,2 +1,140 @@
...
+	}
+
+	refund, err := client.Refund(refundRequest)
+	if err != nil {
+		t.Error("Refund request should not have failed")
+	}
+	if !refund.Success {
+		t.Error("Resulting refund should have been successful")
+	}
+}
```
Comment (yuweihsu01) [inline integration-tests/rocketgate_test.go:140]: nit: space

File: gateways/rocketgate/request_builders.go (hunk 1 above)
Comment (rlcooper46) [reply gateways/rocketgate/request_builders.go:53]: haha realized that when the refund and capture tests were failing. I fixed that

File: rocketgate-sdk/service/service.go [hunk 3]
```diff
@@ -284,7 +284,7 @@ func (r GatewayService) getConnectTimeout(req *request.GatewayRequest) int64 {
 	return int64(connectTimeout) * 1000
 }
 
-func (r GatewayService) getServerNameAndCleanFailedParams(req *request.GatewayRequest, resp *response.GatewayResponse) (string, bool) {
+func (r *GatewayService) getServerNameAndCleanFailedParams(req *request.GatewayRequest, resp *response.GatewayResponse) (string, bool) {
```
Comment (yuweihsu01) [inline rocketgate-sdk/service/service.go:287]: I don't think we need change to pointer for getServerNameAndCleanFailedParams (my test can pass without this change)? I thought pointer is mainly for update ?

Comment (rlcooper46) [reply rocketgate-sdk/service/service.go:287]: i probably left this in while i was trying to find the original bug, ill remove it

**Review by yuweihsu01 — APPROVED** [review]

**Pull request merged**